CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/utilities/VisualGraphUtilities.java

```diff
@@ -99,6 +124,16 @@ public static Vector3f getMixedVertexCoordinates(final GraphReadMethods graph, f
         return coordinates;
     }
 
+    public static Vector3f getMixedVertexCoordinates(final GraphReadMethods graph, final int vertexId, final int xAttribute, final int x2Attribute, final int yAttribute, final int y2Attribute, final int zAttribute, final int z2Attribute, final int cameraAttribute) {
+        final Vector3f coordinates = getVertexCoordinates(graph, vertexId, xAttribute, yAttribute, zAttribute);
+        final Vector3f altCoordinates = getAlternateVertexCoordinates(graph, vertexId, x2Attribute, y2Attribute, z2Attribute);
+        final float mixRatio = getCamera(graph, cameraAttribute).getMixRatio();
+        if (coordinates != null && altCoordinates != null) {
```

Comment:
Neither or these will be null as the functions generating them are always returning new Vector3f objects. Therefore you can probably omit the if statement

Reply:
Fixed